Artifacts/linux-c++ethereum/artifactfile.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2015-01-01/dtlArtifacts.json",
+  "title": "C++Ethereum-Linux",
```

Comment:
Artifacts are automatically filtered by OS type, so you probably don't need to include the "-Linux" suffix.